pkg/identityserver/store/gateway.go

```diff
@@ -63,12 +64,16 @@ type Gateway struct {
 	UpdateLocationFromStatus bool `gorm:"default:false not null"`
 
 	Antennas []GatewayAntenna
+
+	LBSLNSSecret []byte `gorm:"type:BYTEA;column:lbs_lns_secret"`
```

Comment:
New database fields need to be mentioned in `CHANGELOG`, and there needs to be a note about running the schema migration.